packages/metal-component/src/Component.js

```diff
@@ -85,6 +85,12 @@ class Component extends EventEmitter {
 		this.eventsStateKeyHandler_ = null;
 
 		/**
+		 * Whether the element is currently force updating.
```

Comment:
Can we explain what `to be force updating` means a bit more? Not sure if it is common knowledge, but maybe something along the lines, `wether it should ignore `shouldUpdate` and always render...` , something that doesn't include the variable name :)